apparmor.d/profiles-m-r/mutt

```diff
@@ -24,6 +24,42 @@ profile mutt @{exec_path} {
 
   @{exec_path} mr,
 
+  # Used to exec programs defined in the mailcap.
+  # There are countless programs that can be executed from the mailcap.
+  # This profile includes only the most basic.
+  @{bin}/{,ba,da}sh               rix,
+ 
+  @{bin}/sendmail                rPUx,
+  /usr/libexec/sendmail/sendmail rPUx,
```

Comment:
You have `@{lib}` for `/usr/libexec` (the exact path depends of the distribution)

Reply:
Ah nice! I should have checked the abstractions.